Source/gendung.cpp

```diff
@@ -37,8 +37,8 @@ BYTE *pDungeonCels;
 int SpeedFrameTbl[128][16];
 THEME_LOC themeLoc[MAXTHEMES];
 char dPlayer[MAXDUNX][MAXDUNY];
-int dword_5C2FF8;
-int dword_5C2FFC;
+int game_world_width;

```

Comment:
Maybe screen_chunks and screen_blocks. Since it's screen size and not world size.